modules/standard/GMP.chpl

```diff
@@ -424,10 +424,10 @@ module GMP {
 
   pragma "no doc"
   // Initialize GMP to use Chapel's allocator
-  extern proc chpl_gmp_init();
+  private extern proc chpl_gmp_init();
   /* Get an MPZ value stored on another locale */
   pragma "no doc"
-  extern proc chpl_gmp_get_mpz(ref ret:mpz_t,src_local:int,from:__mpz_struct);
+  private extern proc chpl_gmp_get_mpz(ref ret:mpz_t,src_local:int,from:__mpz_struct);
   /* Get a randstate value stored on another locale */
   extern proc chpl_gmp_get_randstate(not_inited_state:gmp_randstate_t, src_locale:int, from:__gmp_randstate_struct);
```

Comment:
is chpl_gmp_get_randstate used in a test? Why can't it be private too? My guess is that all of these chpl_gmp functions should be private... but I wouldn't be surprised if they're used in some tests and it doesn't really matter.

I think chpl_gmp_mpz_nlimbs should probably be private, since it takes an argument with a no-doc'd type.